### ISSUE TITLE
(R) Implement ZipMaker Worker (blocked by #737)

### ISSUE THREAD

**Issue by SaravShah** [sul-dlss/preservation_catalog#707]
- How to create zip file? ( Are using lib zip or shell out ) 
  - Ask Julian or ops what he thinks we should use
  - Zip with no compression
  - Main concern is reproducibility for a zip file.

- Does Zip already exist? 
  - if yes do a unix touch (updates timestamp) - can be implemented later since we aren't cleaning up old zips yet (we think clean up will remove least-recently used first)
  - if no, create a zip
    - and queue the message for zip maker success (already done)
